src/Templates/src/templates/maui-aspire-servicedefaults/Extensions.cs

```diff
@@ -53,16 +53,16 @@ public static TBuilder ConfigureOpenTelemetry<TBuilder>(this TBuilder builder) w
         builder.Services.AddOpenTelemetry()
             .WithMetrics(metrics =>
             {
-                // Enable reporting metrics coming from the .NET MAUI SDK
-                metrics.AddMeter("Microsoft.Maui");
+                // Uncomment the following line to enable reporting metrics coming from the .NET MAUI SDK, this might cause for a lot of added telemetry
```

Comment:
```suggestion
                // Uncomment the following line to enable reporting metrics coming from the .NET MAUI SDK, this might cause a lot of added telemetry
```